set-mongo/src/main/java/eu/europeana/set/mongo/service/PersistentUserSetServiceImpl.java

```diff
@@ -219,13 +218,18 @@ private FindOptions buildMongoPaginationOptions(UserSetQuery query) {
 	    if(query.getItem() != null) {
 		mongoQuery.filter(WebUserSetModelFields.ITEMS + " in", query.getItem());
 	    }
+
+		if(query.getSetId() != null) {
+			mongoQuery.filter(WebUserSetModelFields.IDENTIFIER + " in",   query.getSetId());
```

Comment:
the "in" operator is not needed as the setId is not a list

Reply:
removed "in" parameter